src/hotspot/cpu/ppc/vm_version_ppc.cpp

```diff
@@ -169,6 +169,28 @@ void VM_Version::initialize() {
       FLAG_SET_DEFAULT(UseByteReverseInstructions, false);
     }
   }
+
+  // Power7 and later.
+  if (PowerArchitecturePPC64 > 6) {
```

Comment:
Settings that depend on `PowerArchitecturePPC64` seem to be ordered. You might want to keep it like that.

Reply:
I have moved these flags up. Note that the checks will get removed by [JDK-8331859](https://bugs.openjdk.org/browse/JDK-8331859).